libraries/AP_HAL_ChibiOS/hwdef/AEROFOX-H7/hwdef.dat

```diff
@@ -0,0 +1,237 @@
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_AEROFOX_H7
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 128
+
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# supports upto 8MBits/s
+CANFD_SUPPORTED 8
+
+# with 2M flash we can afford to optimize for speed
+env OPTIMIZE -O2
+
+# system timer
+STM32_ST_USE_TIMER 5
+
+# UART
+SERIAL_ORDER OTG1 UART7 UART4 UART5 USART2 USART1 UART8 USART3 OTG2
+
+#serial1
+PA15 UART7_TX UART7
+PA8 UART7_RX UART7
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_ESCTelemetry
+
+# serial2
+PC10 UART4_TX UART4
+PC11 UART4_RX UART4
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_None
+
+# serial3
+PC12 UART5_TX UART5
+PD2 UART5_RX UART5
+define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_None
+
+# serial4
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_None
+
+# serial5
+PA9 USART1_TX USART1 NODMA
+PA10 USART1_RX USART1 NODMA
+define DEFAULT_SERIAL5_PROTOCOL SerialProtocol_GPS
+
+# serial6
+PE1 UART8_TX UART8
+PE0 UART8_RX UART8
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_RCIN
+
+# serial7
+PD8 USART3_TX USART3
+PD9 USART3_RX USART3
+define DEFAULT_SERIAL7_PROTOCOL SerialProtocol_MAVLink2
+
+# CAN
+PD0  CAN1_RX CAN1
+PD1  CAN1_TX CAN1
+
+PB12 CAN2_RX CAN2
+PB13 CAN2_TX CAN2
+
```

Comment:
```suggestion
# default the 2nd interface to MAVLINK2
define HAL_OTG2_PROTOCOL SerialProtocol_MAVLINK2
```
you removed the SLCAN but did not force the MAVLINK2....recommended, but not a blocker